docs/reference/transfers.md

```diff
@@ -39,9 +39,11 @@ Fields used by each mode of transfer:
 | `flags.balancing_credit`      | optional     | optional | false        | false        |
 | `timestamp`                   | none         | none     | none         | none         |
 
-TigerBeetle uses the same data structures internally and
-externally. This means that sometimes you need to set temporary values
-for fields that TigerBeetle, not you (the user), are responsible.
+TigerBeetle uses the same data structures internally and externally. This means 
+that when creating transfers, you will need to set temporary values for certain 
+fields that TigerBeetle is responsible for. For example, you will set the 
+[`timestamp`](#timestamp) field to `0` when creating a transfer and then
+TigerBeetle will set the field to the timestamp when the transfer was executed.
```

Comment:
Can we DRY this up in one place?

Reply:
Same for other sections below?

Reply:
Does it even make sense to include this note? This feels like something that could just be documented in the client libraries, or the client libraries could hide this fact from users